src/base/random.cc

```diff
@@ -43,7 +43,6 @@
 #include <sstream>
 
 #include "base/logging.hh"
```

Comment:
I suspect you don't need these headers anymore

(logging was to use the panic)

Reply:
Fixed !